examples/utils/scripting_utils.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
`scripting_utils` sounds a bit general to me, shall we just combine `data_utils` and `scripting_utils`?

Reply:
Hmm, meant scripting as in runnable scripts. Things related to arguments, environment, etc. Any suggestions?

I don't think `data_utils` is a good name for a argument handling utility, and I want to avoid any catch all `utils.py` files as an anti-pattern. That leads to bad search-ability, e.g. in an IDE.

Reply:
Curious - what's the issue for searching caused by having on `utils.py`?

Reply:
Think fuzzy file search in a IDE (or codesearch). Start searching "utils.py" and 10 files come up -> not very helpful. In general I've seen a lot of projects try to avoid files with  generic names (within reason), and I find it good practice.